redisvl/extensions/llmcache/semantic.py

```diff
@@ -77,6 +82,9 @@ def __init__(
             [
                 {"name": self.prompt_field_name, "type": "text"},
                 {"name": self.response_field_name, "type": "text"},
+                {"name": self.inserted_at_field_name, "type": "numeric"},
```

Comment:
Checkout how I do the schema encapsulation / wrapping in my routing branch. I think it's the strategy we will want to move towards with all of our interfaces. I am happy to put this in a commit so you can see what it would look like!

Reply:
You mean your `SemanticRouterIndexSchema` class with `from_params()` method?
We can bring that over here